examples/api/test/material/selection_container/selection_container_disabled.0_test.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_api_samples/material/selection_container/selection_container_disabled.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('A SelectionContainer.disabled should disable selections', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.SelectionContainerDisabledExampleApp(),
+    );
+
+    expect(find.widgetWithText(AppBar, 'SelectionContainer.disabled Sample'), findsOne);
+
+    final RenderParagraph paragraph1 = tester.renderObject<RenderParagraph>(find.descendant(of: find.text('Selectable text').first, matching: find.byType(RichText)));
```

Comment:
```suggestion
    final RenderParagraph paragraph1 = tester.renderObject<RenderParagraph>(
      find.descendant(
        of: find.text('Selectable text').first,
        matching: find.byType(RichText),
      ),
    );
```

Reply:
Sure, I've done it in [format: Split long lines](https://github.com/flutter/flutter/pull/156934/commits/2ea58265e7f1f13da330c7dc2df3324783661bd5) :)